src/api/controllers/login_account.py

```
@@ -0,0 +1,178 @@
+from datetime import datetime, timedelta
+from api.models import EateryStore
+from api.datatype.Eatery import EateryID
+import os
+import hashlib
+from django.utils import timezone
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+from django.contrib.auth.hashers import PBKDF2PasswordHasher as handler
+from rest_framework import status
+
+hasher = handler()
+
+def get_user_by_id(id):
+    return EateryStore.objects.filter(id=id.value).first()
+
+def get_user_by_email(email):
+    return EateryStore.objects.filter(email=email).first()
+
+def get_user_by_session_token(session_token):
+    return EateryStore.objects.filter(session_token=session_token).first()
+
+def extract_token(request):
+    auth_header = request.headers.get("Authorization")
+    if auth_header is None:
+        raise Exception("Malformed Request: missing auth header")
+
+    bearer_token = auth_header.replace("Bearer ", "").strip()
+    if bearer_token is None or not bearer_token:
+        raise Exception("Malformed Request: missing bearer header")
```

do you actually want to raise an exception? or do you want to silently return a failure response?